src/main/java/com/wavefront/sdk/jersey/WavefrontJerseyFilter.java

```diff
@@ -292,13 +295,15 @@ public void filter(ContainerRequestContext containerRequestContext,
       wfJerseyReporter.incrementDeltaCounter(new MetricName(responseMetricKey +
           ".aggregated_per_application", aggregatedPerApplicationMap));
 
-     /*
+      /*
        * Overall error response metrics
        * 1) <prefix>.response.errors.aggregated_per_source (Counter)
        * 2) <prefix>.response.errors.aggregated_per_shard (DeltaCounter)
        * 3) <prefix>.response.errors.aggregated_per_service (DeltaCounter)
        * 4) <prefix>.response.errors.aggregated_per_cluster (DeltaCounter)
        * 5) <prefix>.response.errors.aggregated_per_application (DeltaCounter)
+       * API granular error metrics
+       * 1) jersey.server.response.api.v2.alert.summary.GET.errors
```

Comment:
while you are at it
can you replace <prefix> with jersey.server ??
elsewhere in the code?